source/unified-test-format/unified-test-format.rst

```diff
@@ -399,7 +399,7 @@ Tests SHOULD use sequential names based on the entity type (e.g. "session0",
 When defining an entity object in YAML, a `node anchor`_ SHOULD be created on
 the entity's ``id`` key. This anchor will allow the unique name to be referenced
 with an `alias node`_ later in the file (e.g. from another entity or
-`operation`_ document) and also leverage YAML's parser for reference validation.
+`operation`_ object) and also leverage YAML's parser for reference validation.
```

Comment:
I must have missed this when we replaced "document" with "object" throughout the spec a while back. IIRC, @p-mongo brought that up in an old CR and we agreed to only use "document" to refer to MongoDB documents.